pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -33,6 +34,16 @@ import (
 	cm "k8s.io/cloud-provider-vsphere/pkg/common/connectionmanager"
 )
 
+const (
+	// IPv6Family string representation for IPv6
+	IPv6Family = "ipv6"
+	// IPv4Family string representation for IPv4
+	IPv4Family = "ipv4"
+
+	// DefaultIPFamily is the default IP addressing to use for networking
+	DefaultIPFamily string = IPv4Family
```

Comment:
Hi @dvonthenen,

Could remove `string`

Reply:
Will update it!